### DISCUSSION TITLE
Site not working after failed https setup.

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#3263] [Error / Bug Report]
Hi, 

I'm running the latest wiki.js instance on Ubuntu 18.04 and I had a problem last night when tinkering and my site stopped working. I had a backup so made a new droplet on Digital Ocean. The site was working again just now so I decided to setup Lets Encrypt again. This failed due to too many certificates being issued...not a problem as I can do this later. But now my wiki isn't loading at all again. I've restarted it and waited for the jws error message to go but now it's been over a couple of hours and I keep getting a 'unable to connect' or likewise page. 

Thanks
B

**Comment by NGPixel** [comment]
See https://docs.requarks.io/en/troubleshooting#how-to-manually-disable-https-ssl-redirection
to manually disable HTTPS on a non-working wiki
